forge-gui/res/cardsfolder/upcoming/slight_malfunction.txt

```diff
@@ -0,0 +1,8 @@
+Name:Slight Malfunction
+ManaCost:1 R
+Types:Sorcery
+A:SP$ Charm | Choices$ DBDestroyArtifact,DBRoll
+SVar:DBDestroyArtifact:DB$ Destroy | ValidTgts$ Artifact | SpellDescription$ Destroy target artifact.
+SVar:DBRoll:DB$ RollDice | ResultSVar$ X | SubAbility$ DBDamage | SpellDescription$ Roll a six-sided die. When you do, CARDNAME deals 1 damage to each of up to X target creatures, where X is the result.
```

Comment:
reflexive trigger

Reply:
Why resolve, it's currently wrong?

Reply:
Woops seem to have forgot to push my commit.